### PR TITLE
Winrm expansion and cleanup, and a few version reference fixes too

### PR DESCRIPTION
**Pull request by ahgittin**

### CODE REVIEW

**Comment by ahgittin** [comment]
rejigged commit so that you can see the changes i made in the third commit; the second splits up the files (but doesn't change anything else).  so 1st and 3rd commit to be reviewed atomically.

cc @infrastation

File: guide/blueprints/winrm/limitations.md [hunk 1]
```diff
@@ -110,7 +107,7 @@ uploads and in their PowerShell scripts.
 
 ### Windows template settings for an Unattended Installation
 
-Windows template needs certain configuration to be applied to prevent Windows setup UI from being displayed.
+Windows template needs certain configuration to be applied to prevent windows setup UI from being displayed.
```
Comment (infrastation) [inline guide/blueprints/winrm/limitations.md]: It would be better to keep it "Windows".

File: guide/blueprints/winrm/tips.md [hunk 2]
```diff
@@ -104,8 +109,9 @@ config like `pre.install.reboot.required` and `install.reboot.required`. If requ
 installation commands can be split between the pre-install, install and post-install phases
 in order to do a reboot at the appropriate point of the VM setup.
 
-We strongly recommend to **write blueprints in a way that they do NOT restart automatically Windows** and
-use one of the `pre.install.reboot.required` or `install.reboot.required` parameters to perform restart.
+We strongly recommend to **write blueprints in a way that they do NOT restart automatically windows** and
```
Comment (infrastation) [inline guide/blueprints/winrm/tips.md]: "Windows"

File: guide/blueprints/winrm/troubleshoot.md [hunk 3]
```diff
@@ -15,20 +24,26 @@ If the configuration options `userMetadata` or `userMetadataString` are used on
...
 
-### Credentials issue requiring special configuration
+
+### Credentials and privileges requiring special configuration
 
 When a script is run over WinRM over HTTP, the credentials under which the script are run are marked as
 'remote' credentials, which are prohibited from running certain security-related operations. This may prevent certain
 operations. The installer from Microsoft SQL Server is known to fail in this case, for example. For a workaround, please
-refer to [How and Why to re-authenticate withing a PowerShell script](#how-and-why-to-re-authenticate-within-a-powershell-script) 
+refer to [How and Why to re-authenticate withing a PowerShell script](tips.md#how-and-why-to-re-authenticate-within-a-powershell-script)
```
Comment (infrastation) [inline guide/blueprints/winrm/troubleshoot.md]: "within"

**Comment by infrastation** [comment]
The only two links to the new `limitations.md` file are in the middle of other text, maybe you would like to list it in the "learn more" section at the end of `index.md`.

**Comment by ahgittin** [comment]
thanks @infrastation - all addressed merging

**Pull request merged**